portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -85,15 +85,6 @@ function codeAddress(address) {
       alert('Geocode was not successful for the following reason: ' + status);
     }
   });
-  marker.addListener('click', toggleBounce);
-}
-
-function toggleBounce() {
-  if (marker.getAnimation() !== null) {
-    marker.setAnimation(null);
-  } else {
-    marker.setAnimation(google.maps.Animation.BOUNCE);
-  }
```

Comment:
Since I instantiate the map with the drop animation within the codeAddress() function, a helper method isn't exactly necessary. The features still work as I intended without any of those lines, and it prevents lint checks from throwing "unused vars" errors.